projects/data/README.md

```diff
@@ -0,0 +1,67 @@
+# Data
+Scripts for producing training and testing data for `Aframe`
+
+## Environment
+The `data` project environment utilizes `Mamba` and `poetry`. `Mamba` is needed for installing
+the LIGO frame reading libraries [`python-ldas-tools-framecpp`](https://anaconda.org/conda-forge/python-ldas-tools-framecpp/) and [https://anaconda.org/conda-forge/python-nds2-client](`python-nds2-client`), which are unavailable on PyPi.
+
+
+In the root of the `data` project, run 
+```bash
+apptainer build $AFRAME_CONTAINER_ROOT/data.sif apptainer.def
+```
+to build the `data` container.
+
+## Scripts
+The data project consists of four main sub-modules:
+
+1. `data/segments` - Querying science mode segments
+2. `data/fetch` - Fetching strain data
+3. `data/timeslide_waveforms` - Generating waveforms for injection campaigns
+4. `data/waveforms` - Generating waveforms for training Aframe
+
+Additionally, the main executable of each sub-module is exposed via a CLI at `data/cli.py`
+
+## Example: generating training data
+As an example, let's build a training dataset using the CLI in the `data` container we built above
+
+First, let's make a data storage directory, and query science mode segments from [gwosc](gwosc.org)
+```
+mkdir ~/aframe/data/
+apptainer run ~/aframe/images/data.sif \
```

Comment:
Same comment about `$AFRAME_CONTAINER_ROOT` throughout this readme. Could also keep using `~/aframe/images` and just comment that this is an example of where images could be stored

Reply:
done